ledger/sandbox-on-x/reference.conf

```diff
@@ -6,10 +6,21 @@
 # Commented out sections or fields are optional.
 # Some values choosen here as defaults are no intended to be production ready.
 
+# TODO: one of the bigger questions is: parameter vs functional control. in Canton, we are keeping the expert knobs in canton.parameters or canton.participant.p.parameters
+#       and thus separate it from the config options that a user would normally change. at some point, we actually wanted to say that setting parameters is "unsupported".
+#       generally, default parameters should just be chosen such that they never have to be changed or tuned by a normal user.
+#       of course, this isn't 100% clean and consistent in canton so far. but at least, that was the idea.
+#       in the api server config, we do this very much different. the config structure corresponds to the component and the configurations are very close to that.
+
 ledger {
   # The ID of the ledger. This must be the same each time the ledger is started.
   ledger-id = default-ledger-id
 
+  # TODO consolidate with Canton: 
+  #   1. Canton supports multiple reporters concurrently. So you can output to CSV and Prometheus concurrently.
+  #      https://github.com/DACH-NY/canton/blob/main/community/app/src/main/scala/com/digitalasset/canton/metrics/MetricsFactory.scala#L23
+  #   2. Use case classes rather than Strings that have to be parsed. Strings are hard do extend.
```

Comment:
URIs were the ideal mechanism for a traditional CLI, now with HOCON we have the freedom to change that. We will be able to get rid of the URIs once all usages of old CLI are gone,